src/components/Header/index.tsx

```diff
@@ -21,75 +21,77 @@ const menus = [
 const NORMAL_HEIGHT = 42
 const SEARCH_HEIGHT = 95
 
-export default ({ search = true }: { search?: boolean }) => {
+export default ({ search = true, width = window.innerWidth }: { search?: boolean; width?: number }) => {
   const [height, setHeight] = useState(NORMAL_HEIGHT)
 
   useEffect(() => {
     setHeight(NORMAL_HEIGHT)
   }, [setHeight])
-
-  if (isMobile()) {
+  return useMemo(() => {
+    // console.log("normally rerender will not occur with useMemo")
```

Comment:
Should remove ?

Reply:
remove in fulture, now just keep for instruction and test

Reply:
I think we can remove `console.log` and retain annotation